src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -75,9 +75,7 @@ const UsersFragment = ({
   isFirstTimeOpen,
   initialUsers = [],
   followedUsers = [],
-  setFollowedUsers = () => {},
   unfollowedUsers = [],
-  setUnfollowedUsers = () => {},
   setSearchText = () => {},
   setIsFirstTimeOpen = () => {},
   withoutRecent = false,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `setFollowedUsers` and `setUnfollowedUsers` functions from the props might cause issues if these functions are used elsewhere in the component. Please ensure that these functions are not required anymore or have been replaced appropriately.



<!-- This is an auto-generated comment by OSS CodeRabbit -->